src/Interpreters/ProcessList.h

```diff
@@ -48,6 +48,7 @@ enum CancelReason
     UNDEFINED,
     TIMEOUT,
     CANCELLED_BY_USER,
+    CANCELLED_BY_ERROR,
```

Comment:
It means there were parallel processes/threads and some of them failed, so we cancel other processes/threads with this cancel reason.